lib/chezmoi/scripts.go

```diff
@@ -0,0 +1,97 @@
+package chezmoi
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"regexp"
+)
+
+var shebangRegex *regexp.Regexp
+var onceRegex *regexp.Regexp
```

Comment:
Please put these in a `var` block name them `Regexp` (with a `p` at the end) for consistency.

```go
var (
    shebanRegexp = // ...
    onceRegexp   = // ...
)
```